src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx

```diff
@@ -127,7 +156,37 @@ const MissingItemFormEdit = () => {
       dateLost: new Date(formData.dateLost).toISOString() || DateTime.now().toISO(),
       forGuest: false,
     };
+    const originalFields = Object.keys(formData);
+    const newFields = Object.keys(formDataCopy);
+    let newActionNote = '';
+    for (let i = 0; i < originalFields.length; i++) {
+      if (
+        // @ts-ignore
+        JSON.stringify(formDataCopy[originalFields[i]]) != JSON.stringify(formData[newFields[i]])
+      ) {
+        // @ts-ignore
```

Comment:
This ts-ignore appears to be doing nothing, I got no errors when running without it.